### PR TITLE
createBody to be called from add instead of prepare

### PR DESCRIPTION
**Pull request by spydon**
Also updates the version to 0.6.3-rc4

### CODE REVIEW

File: lib/forge2d_game.dart [hunk 1]
```diff
@@ -33,8 +33,8 @@ class Forge2DGame extends BaseGame {
   }
 
   @override
-  void prepare(Component component) {
-    super.prepare(component);
+  Future<void> add(Component component) async {
+    super.add(component);
```
Comment (erickzanardo) [inline lib/forge2d_game.dart]: Shouldn't we await for this before continuing?

Comment (spydon) [reply lib/forge2d_game.dart]: True, good catch! Fixed

**Review by erickzanardo — APPROVED** [review]
LGTM

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
 # CHANGELOG
 
+
+## [0.6.3-rc4]
+ - Changed so that Forge2DGame calls createBody from add instead of prepare
```
Comment (wolfenrain) [inline CHANGELOG.md]: I would personally change this sentence to be more of what the change really is:
```suggestion
 - Renamed the `prepare` method to `add` to be more inline with Flame Game naming.
```

**Review by wolfenrain — APPROVED** [review]
LGTM

**Pull request merged**